onnxruntime/core/util/gemmlowp_common_wrapper.h

```diff
@@ -14,6 +14,10 @@
 #pragma warning(push)
 #pragma warning(disable : 4100)  //'identifier' : unreferenced formal parameter
 #endif
+#ifdef _MSC_VER
```

Comment:
could  it be moved up to the previous ifdef section?

Reply:
yes it can be, any specific reason?

Reply:
Just wondering why they are listed separately, not with the warning 4100.

Reply:
my bad I did not notice the if else in between ... I thought all lines from 8 till 15 were in ifdef __GNUC__
Will make the change